gapis/service/service.proto

```diff
@@ -1299,6 +1299,13 @@ message GetTimestampsRequest {
 // is specified in a TimestampsItem message.
 message Timestamps {
```

Comment:
The request should probably specify whether or not it wants handle mappings at all.
Also if we have a non-0 loop count, I think your handle mappings wont do the right thing, maybe a TBD, but if that is the case then we DEFINITELY should not make it default to on.